forage-android/src/main/java/com/joinforage/forage/android/vault/ForagePinSubmitter.kt

```diff
@@ -109,7 +109,7 @@ internal class ForagePinSubmitter(
         private fun buildVaultUrl(path: String): HttpUrl =
             VAULT_BASE_URL.toHttpUrlOrNull()!!
                 .newBuilder()
-                .scheme("https")
```

Comment:
turns out this wasn't necessary after all and impeded local development against http.

Reply:
Nice!